src/ViewModel/HeroBanner.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace eLife\Patterns\ViewModel;
+
+use Assert\Assertion;
+use eLife\Patterns\ArrayAccessFromProperties;
+use eLife\Patterns\ArrayFromProperties;
+use eLife\Patterns\ViewModel;
+
+final class HeroBanner implements ViewModel
+{
+    use ArrayAccessFromProperties;
+    use ArrayFromProperties;
+
+    private $title;
```

Comment:
There is no way to set the `TitleLength` parameter which is part of the schema for the `hero-banner.mustache` template.